manifests/init.pp

```diff
@@ -430,6 +439,9 @@
   String                                          $default_logging_level     = $logging_level,
   Optional[String]                                $keystore_password         = undef,
   Optional[Stdlib::Absolutepath]                  $keystore_path             = undef,
+  Optional[String]                                $logging_content           = undef,
+  Stdlib::Absolutepath                            $logging_path              = "${configdir}/log4j2.properties",
```

Comment:
Do we really need this path parameter?
The file has to be placed somewhere in the classpath, so does it make sense to let users place it anywhere thy like?